Xamarin.Forms.Platform.iOS/Platform.cs

```diff
@@ -17,7 +17,7 @@ public class Platform : BindableObject, INavigation, IDisposable
 		, IPlatform
 #pragma warning restore
 	{
-		internal static readonly BindableProperty RendererProperty = BindableProperty.CreateAttached("Renderer", typeof(IVisualElementRenderer), typeof(Platform), default(IVisualElementRenderer),
+		public static readonly BindableProperty RendererProperty = BindableProperty.CreateAttached("Renderer", typeof(IVisualElementRenderer), typeof(Platform), default(IVisualElementRenderer),
```

Comment:
I definitely like the spirit of this change but what's your use case here?

Reply:
I'm not 100% sure if it is needed. When I did the original PR for the Popup Control I followed some standard rules I found in other built-in renderers. Here is the usage from the original fork 
https://github.com/ahoefling/Xamarin.Forms/blob/489c149260ce116ac8c68705c45d511e29285c97/Xamarin.Forms.Platform.iOS/Renderers/PopupRenderer.cs#L212-L213

```c#
if (iOS.Platform.GetRenderer(Element) == this)
    Element.ClearValue(iOS.Platform.RendererProperty);
```

As far as I understand the code, it is useful during the `Dispose` invocation to clear the renderer from memory. I can see this being valuable for app performance and keeping the memory allocation small.

If you or other reviewers don't think this is necessary I can omit this code in the Popup Control I am working on for the Xamarin Community Toolkit

Reply:
It should really matter to clear the renderer. That's all connected weakly and it should dispose together.

That being said... I think you could also call this code

Platform.SetRenderer(Element, null);

And it'll have the same effect

Reply:
I took a look at the inner workings of Platform.SetRenderer and it makes sense to me to use your recommendation
```
Platform.SetRenderer(Element, null);
```

I'll update the PR to roll this change out